CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.7.2]
```

Comment:
There are new features added, but they're non-breaking. I've been leaning towards source-breaking changes being `y` releases, and fixes and minor additions being `z` releases.